### PR TITLE
feat: table changes

### PR DESCRIPTION
**Pull request by dheeraja00**

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -213,7 +213,19 @@ class App extends React.Component<{}, State> {
...
+        <TextField
+            componentId="TestName"
+            label="Related Content"
+            placeholder="test-placeholder"
+            value={this.state.appTextCounter}
+            enableTextCounter
+            maxLength={101}
+            minLength={5}
+            // onChange={this.valueUpdater('appTextCounter')}
+          />
```
Comment (ankitsaraogi) [inline example/src/App.tsx]: Remove the comment if not needed

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -375,6 +386,15 @@ class Table extends React.Component<Props, State> {
...
+    const { expandingRowId = [] } = this.props;
+
+    expandingRowId.forEach((item: number) => {
+      this.openNestedRow(item);
+    });
+  }
+
   render () {
     const tableClass = this.getTableClassName();
     // const renderedTable = this.renderChildren();
```
Comment (ankitsaraogi) [inline src/components/Table/Table.tsx]: Remove comment if not needed.

**Comment by dheeraja00** [comment]
@ankitsaraogi completed the required changes

**Comment by ankitsaraogi** [comment]
@dheeraja00 : please complete the changes.

**Comment by dheeraja00** [comment]
@ankitsaraogi Changes are done, please merge

**Comment by dheeraja00** [comment]
@ankitsaraogi I am merging this one ok, because its needed

**Pull request merged**